test/action.yml

```diff
@@ -38,6 +28,10 @@ inputs:
         This is particularly useful when running tests on ephemeral CI instances.
     required: false
     default: 'true'
+  action_repository:
+    default: ${{ github.action_repository }}
+  action_ref:
+    default: ${{ github.action_ref }}
```

Comment:
Please add comments and explain the need for these per https://github.com/MonicaisHer/matter-snap-testing/pull/2#discussion_r1452174671

Reply:
Added